Sources/PointFree/VideoJs.swift

```diff
@@ -10,18 +10,61 @@ var videoJsHead: [ChildOf<Element.Head>] {
       rel(.stylesheet)
       ]),
     .init(script([
+      .init("onload", "videoJsLoaded()"),
       src("https://cdnjs.cloudflare.com/ajax/libs/video.js/7.2.4/video.min.js"),
       `defer`(true)
       ]))
   ]
 
   return [
     style(".vjs-subs-caps-button" % display(.none)),
-    script("window.HELP_IMPROVE_VIDEOJS = false;")
+    .init(node("script", [.text(unsafeUnencodedString("""
```

Comment:
I need to do some interpolating in the script below so gotta go unsafe...

Reply:
Coo. More work to fix in the swift-html branch.